Code/CryMP/Server/SafeWriting/APIImpl.cpp

```diff
@@ -0,0 +1,218 @@
+#include "CryMP/Server/SafeWriting/APIImpl.h"
+
+#include "CryCommon/CryAction/IGameFramework.h"
+#include "CryCommon/CryScriptSystem/IScriptSystem.h"
+#include "CryCommon/CrySystem/ISystem.h"
+#include <Windows.h>
```

Comment:
Ah this evil header here. What about `WinAPI::DLL::Load()` and `WinAPI::DLL::GetSymbol()`?